docs/modules/ROOT/pages/faq.adoc

```diff
@@ -1,72 +1,44 @@
+Defender 1.0 is now deprecated in favor of Defender 2.0. On May 21st 2024, users will no longer have access to Defender 1.0 UI and API. Users that haven't migrated will be forced to migrate. Learn how to migrate xref:migration.adoc[here].
 
-=== Is there any timeline for Defender 2.0 to be generally available?
+== Once I migraete to Defender 2.0, can I continue using Defende 1.0?
 
-Not at this point. We will be collecting feedback from the beta phase and based on the data we’ll collect we will announce our plan to make Defender 2.0 generally available.
+No, once you migrate to Defender 2.0, you will no longer have access to Defender 1.0 UI and API.
 
-=== Are there any breaking changes between Defender 1.0 and Defender 2.0?
+== Will I be able to import my existing Defender configuration to Defender 2.0?
 
-Yes, there are multiple changes to the API endpoints that will require you to update your integrations. You can visit the new API docs and SDK to learn more.
+Yes. All existing configurations for Autotask, Sentinels, etc., is automatically imported to Defender 2.0 under the new features naming convention (described xref:migration.adoc#differences[here]).
 
-=== Will Defender 2.0 offer the same subscriptions as Defender 1.0?
+== Are there any breaking changes between Defender 1.0 and Defender 2.0?
 
-We have yet to finalize the new subscriptions. Once they are finalized we will notify existing users about the expected change.
+Yes, there are multiple changes to the API endpoints that will require you to update your integrations. You can find the API docs https://www.api-docs.defender.openzeppelin.com/#defender-sdk[here, window=_blank].
 
+== What are the pricing options for Defender 2.0?
 
-=== Can I switch back to using Defender 1.0 in case I've migrated to Defender 2.0 beta?
+You can find the pricing for Defender 2.0 https://www.openzeppelin.com/pricing[here, window=_blank]. 
 
-Yes. You can switch back and forth between Defender 2.0 beta to the existing Defender version by clicking on the “Switch back to Defender” option in the navigation menu.
+== Does Defender 2.0 offer support?
 
-image::switch-back-faq.png[Switch back to Defender]
+We offer a Service Level and Support Agreement (SLA) for paid subscriptions. Learn more xref:index.adoc#sla[here].
 
-=== Does Defender 2.0 beta offer the same level of support?
+== How can I get an upgrade my tenant account?
 
-Yes. You can reach out and submit a support request for Defender 2.0 in the same channels as the existing Defender version. Additionally, you can submit feedback or report an issue directly from the web application using the dedicated feedback form.
+You can use the billing page to upgrade your tenant account to a higher tier https://defender.openzeppelin.com/v2/#/billing/[here, window=_blank].
 
-=== How can I get an additional quota or upgrade my tenant account?
+== Where can I see my tier quota usage?
```

Comment:
Nit: we use "tier" and "subscription" interchangeably. Should we stick to one?

Reply:
i see that we are using `subscription` to refer to general paid tiers and `tier` for a specific one, so should be good for now